test/Microsoft.Azure.ServiceBus.UnitTests/API/ApiApprovals.ApproveAzureServiceBus.approved.txt

```diff
@@ -511,6 +526,7 @@ namespace Microsoft.Azure.ServiceBus.Core
         int PrefetchCount { get; set; }
         Microsoft.Azure.ServiceBus.ReceiveMode ReceiveMode { get; }
         System.Threading.Tasks.Task AbandonAsync(string lockToken, System.Collections.Generic.IDictionary<string, object> propertiesToModify = null);
+        System.Threading.Tasks.Task AbandonAsync(System.Collections.Generic.IList<string> lockTokens, System.Collections.Generic.IDictionary<string, object> propertiesToModify = null);
```

Comment:
Attention: breaking change!

If you want to add this to the current major, the change to the contract should be raised as a follow up PR.
Similar to what PR https://github.com/Azure/azure-service-bus-dotnet/pull/486 does.

If the change is added in this PR, this will force a major increase and could delay merge until major version is considered.